cargo-shuttle/src/args.rs

```diff
@@ -131,14 +131,17 @@ pub enum Command {
     Login(LoginArgs),
     /// Log out of the Shuttle platform
     Logout(LogoutArgs),
-    /// Generate shell completions
+    /// Generate shell completions and manpage
     Generate {
         /// Which shell
-        #[arg(short, long, env, default_value_t = Shell::Bash)]
-        shell: Shell,
+        #[arg(short, long)]
+        shell: Option<Shell>,
         /// Output to a file (stdout by default)
         #[arg(short, long, env)]
         output: Option<PathBuf>,
+        /// Generate manpage to the standard output
+        #[arg(short, long)]
+        manpage: bool,
```

Comment:
Since they are mutually exclusive, it feels cleaner if they were separate subcommands: `cargo shuttle generate shell <SHELL>` and `cargo shuttle generate manpage`. Although a breaking change, could this be acceptable? @oddgrd

Reply:
The `OUTPUT` env var usage feels weird as well. Could be a risk of collision with other programs using it. How do we feel about it?

Reply:
Yeah using `OUTPUT` is a bit risky. OTOH although the subcommand approach might look cleaner it would add unnecessary complexity to the code IMO. But I'm fine with either way - and I would say let's have this ready soon and not get caught up into discussion too much 🐻

Reply:
> Since they are mutually exclusive, it feels cleaner if they were separate subcommands: `cargo shuttle generate shell <SHELL>` and `cargo shuttle generate manpage`. Although a breaking change, could this be acceptable?

I think having them as separate subcommands makes the most sense. Regarding the `output` arg, we could remove the env. That makes two breaking changes, but I doubt it will affect many (if any) users.